src/pages/patientView/PatientViewPage.tsx

```diff
@@ -47,6 +47,7 @@ const win:any = (window as any);
 win.patientViewPageStore = patientViewPageStore;
 
 export interface IPatientViewPageProps {
+    params: any; // react route
```

Comment:
so params is automatically passed by ReactRouter?

Reply:
yeah https://github.com/reactjs/react-router-tutorial/tree/master/lessons/06-params